display/tab.go

```diff
@@ -156,12 +156,12 @@ func makeNewTab() *tab {
 			if t.hasContent() {
 				savePath, err := downloadPage(t.page)
 				if err != nil {
-					Error("Download Error", fmt.Sprintf("Error saving page content: %v", err))
+					go Error("Download Error", fmt.Sprintf("Error saving page content: %v", err))
```

Comment:
My problem with this, and all the other `go <modal>` calls in this file is that it opens up the possibility of other things happening before the modal is dismissed. I'm not sure what the solution is though. Maybe all the code in `t.view.SetInputCapture` can be put in a goroutine (with a mutex that blocks multiple versions of it running) that blocks? I'm not sure.

Reply:
This might be a good situation to ask about in #go-nuts on Libera. I wonder if they'd have any advice?

Reply:
Bear in mind the Info() and Error() functions previously just didn't block, but would return as soon as the modal had opened. Using `go Info()` should give similar behaviour to before, the only difference being that now the parent function may return before the modal has even rendered.

But if anything happened immediately *after* it the modals rendered, it would have shown UI breakage before #272, I think. And so we could expect this just to be a return to the previous, non-deadlocking behaviour

Reply:
Fun that we're addressing deadlocking and not-waiting-enough code in one PR :-)
As two sides of one coin.